doc/man3/EVP_PKEY_CTX_ctrl.pod

```diff
@@ -272,16 +272,16 @@ buffer is expected to be the algorithm identifier byte.
 
 The EVP_PKEY_CTX_get_rsa_padding() function gets the RSA padding mode for B<ctx>.
 
-The EVP_PKEY_CTX_set_rsa_pss_saltlen() macro sets the RSA PSS salt length to
-B<len>. As its name implies it is only supported for PSS padding. Three special
-values are supported: B<RSA_PSS_SALTLEN_DIGEST> sets the salt length to the
-digest length, B<RSA_PSS_SALTLEN_MAX> sets the salt length to the maximum
+The EVP_PKEY_CTX_set_rsa_pss_saltlen() function sets the RSA PSS salt length to
+B<saltlen>. As its name implies it is only supported for PSS padding. Three
+special values are supported: B<RSA_PSS_SALTLEN_DIGEST> sets the salt length to
```

Comment:
maybe itemize these values..

Reply:
B<RSA_PSS_SALTLEN_AUTO> causes the salt length to be automatically determined based on the B<PSS> block structure when verifying.

Reply:
Done